dts/bindings/rtc/st,stm32-rtc.yaml

```diff
@@ -10,3 +10,12 @@ include: rtc.yaml
 properties:
   reg:
     required: true
+
+  calibration-out-freq:
+    type: int
+    description: |
+      Calibration output frequency (1 Hz or 512 Hz)
+      0 represents the 512 Hertz, 1 the 1 Hertz.
+    enum:
+      - 0
+      - 1
```

Comment:
Consider making the values passed to the enums clearer by setting it in hertz
```
enum:
  - 1
  - 512
```
and convert the enum to the register value in the driver like
```
#define RTC_CALIB_OUT_FREQ_VAL_1   (0)
#define RTC_CALIB_OUT_FREQ_VAL_512 (1)
```
and expand it like
```
.calibration_out_freq = _CONCAT(RTC_CALIB_OUT_FREQ_VAL_, DT_INST_PROP(0, calibration_out_freq)),
```
See https://github.com/zephyrproject-rtos/zephyr/blob/main/drivers/rtc/rtc_mc146818.c and https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/bindings/rtc/motorola%2Cmc146818.yaml

Reply:
very clearly solution that issue, thank you, i will update right